README.md

```diff
@@ -1 +1,16 @@
-# embedded-berm-bot
\ No newline at end of file
+# embedded-berm-bot
+
+Branch for testing the Arduino code
+
+## MoonDawgArduino
+
+This is the most accurate representation of what the final code will look like
+Make sure to follow the README in the "Libraries" folder to properly run and compile the file in the Arduino IDE
+
+## MoonDawgArduinoTesting
+
+This is outdated as of now. The classes aren't made correctly
+
+## Bot_Passthrough_Code
+
+This is the code that controls the bot through an RC controller
```

Comment:
i dont see this in the code

Reply:
This must be carry over from the code in the reference branch